### PR TITLE
hardening/3rd-party-prs

### PR DESCRIPTION
**Pull request by kzangeli**
Added the label "pull_request" in github actions for:
* func-tests
* unit-tests and
*  mintaka-compat-tests

Those three only had the label "push".

Hopefully, this fixes the problem with not running the tests for pull requests from forks.

And if not ... please help!!!

### CODE REVIEW

File: .github/workflows/unit.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: Unit Test
 
 on:
   push:
+pull_request:  pull_request:
```
Comment (rschwebel) [inline .github/workflows/unit.yml]: wrong indentation

**Review by rschwebel — COMMENTED** [review]
Apart from the wrong indentation I commented inline, I think this is the right thing to do.

File: .github/workflows/unit.yml (hunk 1 above)
Comment (kzangeli) [reply .github/workflows/unit.yml]: Not sure what happened there. Seems like I "yanked" twice without noticing.
Fixing it the "way I understood it", as in the other two ymls.

Apart from that, I see I'm getting double runs for functests now (push + pull_request).
Makes sense, but that's not really desirable,
I'll remove the "push" label and only trigger functests (et al) for pull_request

Comment (kzangeli) [reply .github/workflows/unit.yml]: Also, if the following:
```
  pull_request:
    branches:
      - develop
```
means we only run the compliance test for develop (PRs aiming at develop I must understand that), then, why not remove that restriction?
The compliance test takes like a minute and it doesn't hurt.

**Pull request merged**